src/api/consumer/lib/switch.ts

```diff
@@ -26,51 +26,55 @@ export default async function switchAction(
 
 async function populateSwitchProps(consumer: Consumer, switchProps: SwitchProps) {
   const lanes = await consumer.scope.listLanes();
-  const { laneName, remoteScope } = switchProps;
-  if (remoteScope) {
+  const localLane = lanes.find((lane) => lane.name === switchProps.laneName);
+
+  if (!localLane) {
```

Comment:
I'd suggest switching the if/else. (`if (localLane) {} else {}`).

Reply:
fixed